test/accessors/SimulateTxAccessor.spec.ts

```diff
@@ -74,10 +81,17 @@ describe("SimulateTxAccessor", async () => {
                 userBalance.add(parseEther("1")),
             );
             expect(simulation.success).to.be.true;
-            expect(simulation.estimate.toNumber()).to.be.lte(15000);
+            expect(simulation.estimate.toNumber()).to.be.lte(hre.network.zksync ? 30000 : 15000);
         });
 
-        it("simulate selfdestruct", async () => {
+        it("simulate selfdestruct", async function () {
```

Comment:
All other tests uses arrow function expressions, please adjust this test and make sure it's consistent across the updated code (I was some other test files where this was changed)

Reply:
Arrow function has no own context. Function expression was changed to enable such features as [inclusive tests](https://mochajs.org/#inclusive-tests)

Reply:
Ah, I see. But it seems there are some places where it wasn't changed to enable anything? Like https://github.com/safe-global/safe-contracts/pull/625/files#diff-291a4add941d879c99b79d09eed2c4206c6f23edd4f5edbc93d2bdd18e1c91deR110-R153

Reply:
oops, indeed
I will revert them back :wink: 

Some details:
There are some places where we skipped the tests because of `.send` function usage, it was not properly supported by zksync. However, you have already changed it `.call`. Therefore, we removed the conditional skipping. But this is a leftover...